re2.yaml

```diff
@@ -62,6 +62,9 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: re2 dev
+    dependencies:
+      runtime:
+        - re2
```

Comment:
Why not fix the permissions on the re2 library?

Reply:
how?